src/index.ts

```diff
@@ -2,6 +2,7 @@ import { DownloadCard, HorizontalImageCard, ImageAndDetailsCard, RichContentCard
 import { Container } from "./components/container/Container";
 import { Breadcrumbs } from "./components/denhaag-wrappers/breadcrumbs/Breadcrumbs";
 import { EditableTableRow } from "./components/editableTableRow/EditableTableRow";
+import { SelectMultiple, SelectSingle, ISelectValue } from "./components/formFields/select/select";
```

Comment:
These can best be imported via `from "./components/formFields";`, like all other formFields.